testsuite/tests/kuadrant/authorino/conftest.py

```diff
@@ -1,18 +1,21 @@
 """Conftest for Authorino tests"""
 import pytest
+from weakget import weakget
 
 from testsuite.openshift.objects.auth_config import AuthConfig
 from testsuite.objects import Authorino, Authorization
 from testsuite.openshift.objects.authorino import AuthorinoCR
 
 
 @pytest.fixture(scope="session")
-def authorino(authorino, openshift, blame, request) -> Authorino:
+def authorino(authorino, openshift, blame, request, testconfig) -> Authorino:
     """Authorino instance"""
     if authorino:
         return authorino
 
-    authorino = AuthorinoCR.create_instance(openshift, blame("authorino"))
+    authorino = AuthorinoCR.create_instance(openshift,
```

Comment:
There should be a timeout for the `wait_for_ready`, so it should not wait infinitely long (not sure how long it does wait tho). Try-catch won't help as I don't think it will throw an error, as the CR should be correctly deployed, only deployment will keep failing.